### PR TITLE
docs: make getting started button clearer

### PR DESCRIPTION
**Pull request by MichaelDrogalis**
### Description 

Can't hurt to make it easier to start. :)

Screen Shot 2021-05-12 at 2 11 53 PM

### CODE REVIEW

**Review by colinhicks — APPROVED** [review]

File: docs/index.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ keywords: ksqldb
 [ksqlDB](https://ksqldb.io/) is a database purpose-built
 for stream processing applications on top of {{ site.aktm }}.
 
+[Get Started](https://ksqldb.io/quickstart.html){ .md-button .md-button--top-level }
```
Comment (agavra) [inline docs/index.md:12]: are we calling it `md-button` in honor of you? 😆

**Review by agavra — APPROVED** [review]
Thanks for the quick turn around!

File: docs/index.md (hunk 1 above)
Comment (MichaelDrogalis) [reply docs/index.md:12]: Ahah

**Comment by MichaelDrogalis** [comment]
@JimGalasyn Could we break the norm and proactively merge this into the live branch? Just the current version is fine.

**Comment by JimGalasyn** [comment]
@MichaelDrogalis  Will do! Also will cherry-pick to 0.18.x-ksqdb so the fix is there, too.

**Comment by JimGalasyn** [comment]
@MichaelDrogalis kk this puppy is live now.

**Pull request merged**